modules/otel-collector-components.adoc

```diff
@@ -871,6 +871,48 @@ This extension supports traces, metrics, and logs.
 <12> You can assign the authenticator configuration to an OTLP exporter.
 
 
+[id="oidcauth-extension_{context}"]
+=== OIDC Auth Extension
+
+:FeatureName: The OIDC Auth Extension
+include::snippets/technology-preview.adoc[]
+
+The OIDC Auth Extension authenticates incoming requests to receivers by using the OpenID Connect (OIDC) protocol.
+It validates the ID token in the authorization header against the issuer and updates the authentication context of the incoming request.
+
+.OpenTelemetry Collector custom resource with the configured OIDC Auth Extension
+[source,yaml]
+----
+  config: |
+    extensions:
+      oidc:
+        attribute: authorization # <1>
+        issuer_url: https://example.com/auth/realms/opentelemetry # <2>
+        issuer_ca_path: /var/run/tls/issuer.pem # <3>
+        audience: otel-collector # <4>
+        username_claim: email # <5>
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            auth:
+              authenticator: oidc
+    exporters:
+      otlp:
+        endpoint: <endpoint>
+    service:
+      extensions: [oidc]
+      pipelines:
+        traces:
+          receivers: [otlp]
+          exporters: [otlp]
+----
+<1> The name of the header that contains the ID token. The default name is `authorization`.
+<2> The base URL of the OIDC provider.
+<3> This field is optional. The path to the issuer's CA certificate.
```

Comment:
```adoc
<3> (Optional) The path to the issuer's CA certificate.
```

Reply:
@kowen-rh, oh hello, Kevin, long time no see! I'm just getting to this PR.

Thank you.

I rewrote it as follows:

`<3> Optional: The path to the issuer's CA certificate.`